src/itdelatrisu/opsu/audio/SoundController.java

```diff
@@ -311,6 +311,15 @@ private static void playClip(MultiClip clip, float volume, LineListener listener
 			}
 		}
 	}
+	
+	//TODO Workaround for applause/failsound not stopping
+	/**
+	 * Stops playing a sound, if it is currently active
+	 * @param s The sound effect
+	 */
+	public static void stopSound(SoundComponent s) {
```

Comment:
For usage: You can just call this method for applause/fail on the `leave()` event in game states where they're played, instead of the places you're currently calling this.